docs/Manifests.md

```diff
@@ -1,6 +1,190 @@
 # Manifests
 
-## AppManifest
+## AppManifest Specification
+
+The AppManifest YAML file defines your application's components, routes, resources, and lifecycle events.
```

Comment:
Maybe provide details here about where the AppManifest file needs to exist within your source code to be registered by Noop? I.e. `./noop/app.yml`.

Reply:
Yes very good point. Thank you will add.